Gems/Prefab/PrefabBuilder/PrefabGroup/PrefabGroupBehavior.cpp

```diff
@@ -203,7 +203,7 @@ namespace AZ::SceneAPI::Behaviors
 
         if (!manifestUpdates)
         {
-            return Events::ProcessingResult::Failure;
+            return Events::ProcessingResult::Ignored;
```

Comment:
This means that the pipeline has been configured to generate default proc prefabs and the scene has a mesh, but the system does not have enough information (manifest updates) in order to generate a default procedural prefab. So ignoring this situation might be confusing. 

Maybe add an AZ_Warning and return ignored would be better?

Reply:
Should the warning contain a sort of suggestion on what more needs to be added i.e. Maybe an IMeshData or ITransform node? Or just a warning that things might not work as expected?

Reply:
The Scene Graph is sensitive to having a IMeshData in the scene, so having a warning that suggests adding at least one mesh data node makes sense to me.

Reply:
I've added the warning in my latest commit